spec_tests/spec/app-staging/apparmor_spec.rb

```diff
@@ -25,6 +25,35 @@
   end
 end
 
+# declare expected app-armor capabilities for apache2
+apache2_capabilities = %w(
+  dac_override
```

Comment:
I think we got rid of this capability as of the changes in `release/0.3.4` (which have yet to be merged into the develop branch).